tests/planner/unit/test_prophet_predictor.py

```
@@ -0,0 +1,152 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for ProphetPredictor — focusing on timestamp alignment between
+add_data_point() and predict_next().
+
+Regression test for the bug where predict_next() computed:
+    next_timestamp = start_date + timedelta(seconds=curr_step)
+instead of:
+    next_timestamp = start_date + timedelta(seconds=curr_step * step_size)
+
+With the default step_size=180 this meant the forecast target was 180× too
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)